pandas/core/arrays/sparse.py

```diff
@@ -2014,11 +2014,15 @@ def from_coo(cls, A, dense_index=False):
         from pandas.core.sparse.scipy_sparse import _coo_to_sparse_series
         from pandas import Series
 
-        result = _coo_to_sparse_series(A, dense_index=dense_index,
-                                       sparse_series=False)
-        result = Series(result.array, index=result.index, copy=False)
+        try:
+            result = _coo_to_sparse_series(A, dense_index=dense_index,
+                                           sparse_series=False)
+            result = Series(result.array, index=result.index, copy=False)
```

Comment:
you want to do this inside _coo_to_sparse

Reply:
Done!